main.ts

```diff
@@ -44,14 +44,63 @@ function createWindow() {
 
 }
 
+// Edited version of a template menu-bar provided by the electron API,
+// refer to https://electronjs.org/docs/api/menu for more information.
+const mainMenuTemplate: Electron.MenuItemConstructorOptions[] = [
```

Comment:
I am not sure how adding a menuBar will resolve #47 tho. 

Initially I thought it could be a bug in https://github.com/testathor/CATcher/blob/master/src/app/core/directives/ctrl-key.directive.ts

I don't have a Mac to test this. @RonakLakhotia, can help to test this?

Reply:
Will look into this

Reply:
I referred to this [https://pracucci.com/atom-electron-enable-copy-and-paste.html](article) that mentioned that the cause of the unregistered keystrokes for MacOS was due to the lack of the applications menu with keybinding to the clipboard.

Also the Windows Version of the app seemed to work with other shortcuts such as Ctrl+A (to select text in a textbox for example) worked while the same shortcut (Cmd+A) was unresponsive. The https://github.com/testathor/CATcher/blob/master/src/app/core/directives/ctrl-key.directive.ts didn't explicitly list support for the `Ctrl+A` shortcut hence I presumed it was the Menu Keybinding issue.

Reply:
Hmm, this seems to be a regression after electron packages the app. Refer to [this](https://github.com/electron/electron/issues/2591). Your solution is an acceptable workaround. Have you tested it thoroughly? Can you post a working GIF?

Reply:
Yes the [article](https://pracucci.com/atom-electron-enable-copy-and-paste.html)'s method does seem to be a regression, however, the implementation I used makes use of the inbuilt `roles` in electron as seen in the electron [API](https://electronjs.org/docs/api/menu). So in this case I am not manually listing the shortcut keys. Sure, i'll create a few test gifs and put them up here in abit.

Reply:
## A bunch of keys tested:
![SomeKeysEditIssue](https://user-images.githubusercontent.com/39243082/57834588-344f1d80-77ef-11e9-9d79-09a798521acc.gif)

Reply:
## Current Menu Implementation
![WorkingMenu](https://user-images.githubusercontent.com/39243082/57834901-eab30280-77ef-11e9-8dda-043765e5abfe.gif)

Reply:
## Testing in new Issue Component
![KeysNewIssue](https://user-images.githubusercontent.com/39243082/57835028-47aeb880-77f0-11e9-8986-dbb34241a264.gif)

Reply:
are there any regressions in the windows environment?

Reply:
Yes, just tested it on my Desktop PC (Windows), Tested the same commands as I did for the mac, no issues.